### DISCUSSION TITLE
Anyone try the new ASIO4ALL 2.15 beta?

### DISCUSSION THREAD

**Discussion by crossrhythm** [jamulussoftware/jamulus#1233] [General]
Hi,
It looks like a new ASIO4ALL beta is available, version 2.15.  https://www.asio4all.org; direct link: https://www.asio4all.org/downloads_11/ASIO4ALL_2_15(Beta1)_English.exe 

I'm on a Mac so can't try it but would love to know how it works... should recommend it to others who rely on it for connection to Jamulus?  

Thanks!

**Comment by dcorson-ticino-com** [comment]
I just tried the 2.15 beta knowing that I could never get 2.14 to work on my Lenovo.
As with the old version I get just buzzing noises and some time later a blue screen from Windows.
But I have heard that others say it is an improvement.

**Comment by ann0see** [comment]
I tried it and at least for me, it now even detects the correct in/outputs. So I'd say it's an improvement.
